### DISCUSSION TITLE
分享点自己拍的照片

### DISCUSSION THREAD

**Discussion by WFGoose** [share121/inter-knot#2644] [其他]
业余爱好者，白天去光映广场遛弯随便拍的。（ps：有人知道茶奶店的店员小姐姐叫什么吗？有敲敲账号吗？蹲蹲）
![20240816003608](https://github.com/user-attachments/assets/fbb485fe-4c4d-4317-8845-ff9dc99be635)
![20240816003904](https://github.com/user-attachments/assets/b61f55d5-d6fb-405e-adbb-2555d267ebf5)
![20240816004416](https://github.com/user-attachments/assets/ef5eb2a9-171f-4b3a-82fa-5f8897b297dd)
![20240816010517](https://github.com/user-attachments/assets/82afe310-7f6e-4982-984e-e3b0ecb49bdd)
![20240816003711](https://github.com/user-attachments/assets/a3a3fb4b-c1e2-4544-9be0-de37c6d853de)
![20240816004023](https://github.com/user-attachments/assets/c9e5cbaf-c0c2-4a4d-87b7-ccf4ad871963)

**Comment by wufang994** [comment]
![20240704122430](https://github.com/user-attachments/assets/1ff8f154-d210-4f20-aedf-92d9461fb2c3)
